doc/tutorials/cpu_sharing.rst

```diff
@@ -0,0 +1,252 @@
+.. _cpu_sharing:
+
+ACRN CPU Sharing
+################
+
+Introduction
+************
+ 
+The goal of CPU Sharing is to fully utilize the physical CPU resource, to support more virtual machines. Current ACRN only supports partition mode, which is a strict 1 to 1 mapping mode between vCPUs and pCPUs. Because of the lack of CPU sharing ability, the number of VM is limited. To support the CPU Sharing, we introduce a schedule framework and implement two simple tiny scheduling algorithms to satisfy embedded device requirements.
```

Comment:
Is CPU sharing available to all types of VMs?

I think it would be good to mention if there are restrictions there or cases where CPU sharing is **not** availabe (e.g. pre-launched User VM, RTVM, etc.)

Reply:
Thanks for your comments.
Only the VM with lapic passthrough is not supported.